src/status.ts

```diff
@@ -15,9 +15,22 @@ class StatusBar {
     private item: vscode.StatusBarItem;
     timeout: NodeJS.Timer | null = null;
     counting = false;
+    userColorCustomizations: any;
+    gitDuckColorCustomizations: any;
 
     constructor() {
+        const statusBarFGColor = '#802525';
+        const statusBarBGColor = '#F85246';
         this.item = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Right);
+        this.userColorCustomizations = workbenchConfig.get('colorCustomizations');
```

Comment:
I'd store the `userColorCustomizations` in the `start()` method (which is called when the user starts recording). Else, users may still lose color scheme changes if changed with the GitDuck extension installed because we'd restore the outdated `userColorCustomizations`.